eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -42,9 +42,12 @@ module Cardano.Ledger.Conway.Governance (
   indexedGovProps,
   Constitution (..),
   ConwayEraGov (..),
-  thresholdSPO,
-  thresholdDRep,
-  thresholdCC,
+  getStakePoolThreshold,
+  getDRepThreshold,
+  getCommitteeThreshold,
```

Comment:
I don't think that prepending `get` is necessary

Reply:
Agreed. Renamed to `voting[Entity]Threshod`